setup.py

```diff
@@ -32,7 +32,6 @@
         "Programming Language :: Python :: 3.10",
         "Framework :: Django",
         "Framework :: Django :: 2.2",
-        "Framework :: Django :: 3.1",
         "Framework :: Django :: 3.2",
```

Comment:
You can add the django 4.0 classifier ;).

Reply:
Added!